scripts/spack/configs/toss_4_x86_64_ib/spack.yaml

```diff
@@ -298,11 +298,11 @@ spack:
       - spec: cppcheck@2.9
         prefix: /collab/usr/gapps/axom/devtools/toss_4_x86_64_ib/latest/cppcheck-2.9
     doxygen:
-      version: [1.8.14]
+      version: [1.9.6]
```

Comment:
@bmhan12 -- was the doxygen upgrade necessary or just maintenance?

Reply:
> @bmhan12 -- was the doxygen upgrade necessary or just maintenance?
 
Maintenance-ish. Spack's highest default version changed for doxygen, so this bump was necessary for host-config setup.